### PR TITLE
Properly reading parameters with whitespaces from IR 

### PR DESCRIPTION
**Pull request by artkuli**
### Tickets:
 - *83272*

### CODE REVIEW

File: src/bindings/python/tests/test_runtime/test_core.py [hunk 1]
```diff
@@ -429,3 +429,72 @@ def test_infer_new_request_return_type(device):
     assert arr.shape == (10,)
     assert arr.dtype == "float32"
     assert arr.nbytes == 40
+
+
+def test_read_model_node_names_with_spaces():
```
Comment (ilyachur) [inline src/bindings/python/tests/test_runtime/test_core.py:434]: @artkuli Why did you add a python test?
I think it should be a C++ test.

Moreover can we add an additional test when we add output in the runtime?

File: src/frontends/ir/src/utils.cpp [hunk 2]
```diff
@@ -113,4 +113,15 @@ void str_to_set_of_strings(const std::string& value, std::set<std::string>& res)
     }
 }
 
+void str_to_container(const std::string& value, std::vector<std::string>& res) {
```
Comment (ilyachur) [inline src/frontends/ir/src/utils.cpp:116]: I don't see the code which uses this method :)

File: src/bindings/python/tests/test_runtime/test_core.py (hunk 1 above)
Comment (artkuli) [reply src/bindings/python/tests/test_runtime/test_core.py:434]: @ilyachur The problem was related to the reading parameters from IR not directly to adding outputs in the runtime. The `std::stringstream` inside template method `str_to_container` cut the spaces in names so I wrote specifier method with different behavior when we would like to cast string to the `std::vector<std::string>`. I added Python test because in my opinion, in this case the bigger overview is better but obviously I can add also the test on C++ side 😊

Comment (artkuli) [reply src/bindings/python/tests/test_runtime/test_core.py:434]: @ilyachur  C++ tests added

Comment (ilyachur) [inline src/bindings/python/tests/test_runtime/test_core.py:434]: I think we can remove this test because we cover all changes on C++ level :)

**Review by ilyachur — APPROVED** [review]

**Review by l-bat — APPROVED** [review]
Thank you!

File: src/bindings/python/tests/test_runtime/test_core.py (hunk 1 above)
Comment (artkuli) [reply src/bindings/python/tests/test_runtime/test_core.py]: Test removed :)

**Review by tomdol — APPROVED** [review]
